spec/requests/messages_request_spec.rb

```diff
@@ -17,6 +17,22 @@
       end
     end
 
+    describe 'authentication and authorization' do
+      let(:message) { create(:message, organization: organization) }
+
+      it 'renders unauthorized if not logged in' do
+        patch(organization_message_highlight_url(message, organization_id: message.organization_id, format: :json), params: params)
+        expect(response).to be_unauthorized
+      end
+
+      it 'renders not found for other users' do
+        patch(organization_message_highlight_url(message, organization_id: message.organization_id, format: :json, as: create(:user)),
+              params: params)
+        puts response.code
+        expect(response).to be_not_found
+      end
+    end
+
```

Comment:
you added a shared example in the dashboard controllers PR, then in subsequent PRs here and https://github.com/tactilenews/100eyes/pull/1987/files#diff-b1db8ca954f27e6686bdee8dd73430451cca3d09ff0e6abfc8ba3b3c67b3eafdR80-R84, you decided not to use them.
Why?

Reply:
Well yes, good question. They are not very usable as on the first case it's a JSON response with other behavior and expectations. Other case is not about users at all. Maybe the shared specs are DRYed too early.. let's see if they are of use at other place.. if not I'm going to remove them.